src/lib.rs

```diff
@@ -69,7 +69,12 @@ pub type BigRational = Ratio<BigInt>;
 
 /// These method are `const` for Rust 1.31 and later.
 impl<T> Ratio<T> {
-    /// Creates a `Ratio` without checking for `denom == 0` or reducing.
+    /**
```

Comment:
I'm fine with the panic documentation, but please don't change the comment style.

Reply:
If one commenting style should be used over the other, this should be mentioned in some coding guidelines. I haven't seen anything like that mentioned in README.md, nor was the style change caught with the CI. Perhaps this could be improved.

Regarding the change: I've used the comment style that is simply less painful to use. Feel free to change this back if you feel that it's not acceptable. Or maybe at some later point I'll come back to this PR and make the change myself.

Reply:
I suppose rustfmt's [`normalize_comments`](https://rust-lang.github.io/rustfmt/?version=master&search=#normalize_comments) might cover this, but that's an unstable option. Take note what is considered "normal" though, to the extent that rustfmt is prescriptive.

In the absence of explicit style guidelines, I would suggest sticking with the prevailing style when contributing to any project. The `/**` style for documentation is not used anywhere in this project, and you actively changed existing comments. I might be more forgiving for brand new comments, but changes should stick to their purpose.